src/index.html

```diff
@@ -1,12 +1,441 @@
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,500;1,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          alt="logo" 
+          class="logo-site">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="is-active nav__link">
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samrtphoneAndPhone" class="nav__link">
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#laptopAndPC" 
+              class="nav__link" 
+              data-qa="nav-hover"
+              >LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablet" class="nav__link">
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#foto" class="nav__link">
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              VIDEO
+            </a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+    
+    <main class="main-content">
+      <article class="container-section">
+        
+        <section class="section">
+          <div class="card">
+            <div class="card__item" data-qa="card">
+              <img 
+                src="#" 
+                alt="photo-imac" 
+                class="card__image">
+              <h3 class="card__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h3>
+              <p 
+                class="card__product-code"
+                >Product code: 195434
+              </p>
+              <div class="feedback">
+                <div class="list-stars">
+
+                  <div  class="list-stars__star list-stars__star--active">
+                  </div>
+
+                  <div  class="list-stars__star list-stars__star--active">
+                  </div>
```

Comment:
Don't add a modifier to every star. Add modifier to parent block
```suggestion
                  <div  class="list-stars__star" />
```